docs/ways-you-can-help-us.md

```diff
@@ -66,6 +66,33 @@ open-source firmware mission. It is straightforward if you don't have time for
 other contributions, and there are multiple options - feel free to choose one
 that works for you.
 
+### Become a Dasharo Supporter
+
+Buy the Dasharo Supporters Entrance subscription if you own/use/buy a machine
+running Dasharo firmware:
+
+* [1-year subcription](https://3mdeb.com/shop/open-source-hardware/subscription-for-dasharo-compatible-with-msi-pro-z-690a-ddr4/)
```

Comment:
Should we not have options just for Dasharo SE for 1/2/3 years without `compatible with MSI Pro Z690-A` ? In case there will be more options (like e.g. with stable releases for OptiPlex or T1650).

We had got so many ideas (Dasharo Patreon/ CE/ SE/ OEM/ Subscription) that one might get lost. We should probably pick one and try to stay with it.

Reply:
Agree, we should generalize that, but key problem is that subscription most likely make sense for given platform price point.

e.g. it may make sense to add subscription of 60EUR for platform that cost 1k EUR, but does it make sense for platform that cost 100-300EUR?

> We had got so many ideas (Dasharo Patreon/ CE/ SE/ OEM/ Subscription) that one might get lost. We should probably pick one and try to stay with it.

Agree, but do we know what will work? We change things because we learn all the time and we hope for continues improvements. I believe we building better options then those before. We should have clear distinction between SE buyers and donors through Patreon membership (let's be honest Patreon names were invented because you are forced to do that when you create Patreon). Some community members thought Patreon would be much better then subscription and we still get such feedback e.g. in PC Engines survey.

To sum up, I'm all about one option because I know we need focus, but @TomaszAIR do you have idea what that would be? What will work?

So maybe at this point we should at least make distinction clear and if community and customers don't like it they will tell us.

Reply:
> e.g. it may make sense to add subscription of 60EUR for platform that cost 1k EUR, but does it make sense for platform that cost 100-300EUR?

Agree, I did not think about that earlier.

> Some community members thought Patreon would be much better then subscription and we still get such feedback e.g. in PC Engines survey.

I did not know about that aspect. I hope we will get some brainstorming sessions about DTS/Dasharo/subscription model and will be able to present clear news on DUG#1.

Reply:
Yes, I also hope for that, but there is very little time for providing clear solution. We slowly getting there. Let's assign Dasharo Subscription product generalization for all supported hardware platforms. Internally we should count how many subscriptions we sold and what do we release we should communicate during DUGs.